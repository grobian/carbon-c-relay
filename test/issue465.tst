host.b0123456.component1.c
host.b0123456.component1.c_rate
