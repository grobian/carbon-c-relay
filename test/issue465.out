listen
    type linemode
        2003 proto tcp
        2003 proto udp
        /tmp/.s.carbon-c-relay.2003 proto unix
    ;

statistics
    submit every 60 seconds
    prefix with carbon.relays.test_hostname
    ;


aggregate host\.b[0-9]+\.(.+)\.c
    every 60 seconds
    expire after 120 seconds
    timestamp at start of bucket
    compute average write to
        aggregation.\\1.c
    ;
aggregate host\.b[0-9]+\.component1\.c
    every 60 seconds
    expire after 120 seconds
    timestamp at start of bucket
    compute average write to
        aggregation.component1.c
    ;

aggregation
    host\.b[0-9]+\.(.+)\.c (regex) -> host.b0123456.component1.c
    average(aggregation.\\1.c) -> aggregation.component1.c 
aggregation
    host\.b[0-9]+\.component1\.c (regex) -> host.b0123456.component1.c
    average -> aggregation.component1.c 
aggregation
    host\.b[0-9]+\.(.+)\.c (regex) -> host.b0123456.component1.c_rate
    average(aggregation.\\1.c) -> aggregation.component1.c_rate 
aggregation
    host\.b[0-9]+\.component1\.c (regex) -> host.b0123456.component1.c_rate
    average -> aggregation.component1.c_rate 
